src/chrome/content/rules/TikTok.com.xml

```diff
@@ -0,0 +1,53 @@
+<!--
+	Nonfunctional hosts in *.tiktok.com:
+		Mismatch:
+			+ tiktok.com
+
+	CDN subdomains are frequently changing.
+-->
+<ruleset name="Tiktok.com">
+	<target host="tiktok.com" />
+	<target host="www.tiktok.com" />
+	<target host="activity.tiktok.com" />
+	<target host="ads.tiktok.com" />
+	<target host="careers.tiktok.com" />
+	<target host="creatormarketplace.tiktok.com" />
+	<target host="datahub.tiktok.com" />
+	<target host="developers.tiktok.com" />
+	<target host="getstarted.tiktok.com" />
+	<target host="m.tiktok.com" />
+	<target host="newsroom.tiktok.com" />
+	<target host="support.tiktok.com" />
+	<target host="t.tiktok.com" />
+	<target host="vm.tiktok.com" />
+	<target host="vt.tiktok.com" />
+
+	<!-- CDNs: -->
+	<target host="*.tiktokcdn.com" />
```

Comment:
It might be good to add a "real" test URL, with some content. What do you think?